src/Footer.tsx

```diff
@@ -0,0 +1,42 @@
+const Footer = () => {
+    return (
+    <footer className = "footer pt-24 pb-3 flex flex-col text-white fixed bottom-0 left-0 w-screen md:flex-col-reverse md:justify-between md:flex-wrap-reverse">
```

Comment:
I changed the mobile footer layout a bit, sorry

<img width="461" alt="image" src="https://github.com/webimpactuw/poke-bay/assets/25410985/e2dba835-85d3-4496-9321-3cc0a2a3feb8">